src/components/GetArticle.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "../api";
+
+export const GetArticle = () => {
+  const [loading, setLoading] = useState(true);
+  const { article_id } = useParams();
+
+  const [article, setArticle] = useState([]);
+
+  useEffect(() => {
+    getArticleById(article_id).then((result) => {
+      setArticle(result);
+      setLoading(false);
+    });
+  }, []);
+  if (loading) {
+    return <p className="loading">loading</p>;
+  }
+  return (
+    <section>
+      <h2 className="idTitle" key={article.title}>
```

Comment:
This would be considerably DRY-er if you de-structured the values off of article above your return.